authors/apps/articles/serializers.py

```diff
@@ -5,10 +5,40 @@
 
 
 class ArticleSerializer(serializers.ModelSerializer):
+    """Serializer for articles."""
+    favourite = serializers.SerializerMethodField(method_name='get_favorite')
+    favouritesCount = serializers.SerializerMethodField(
+        method_name='get_favorites_count')
 
     class Meta:
+        """Declare all fields to be returned from the model of articles."""
+
         model = Article
-        """
-        Declare all fields we need to be returned from the model of articles
-        """
-        fields = '__all__'
+        fields = (
```

Comment:
@Bileonaire It would be cleaner if you used the `exclude` attribute in meta.

Reply:
@parseendavid I am adding onto all the fields that are in the database. I have defined the fields I am adding in the article serializer class before the meta class